src/Http/Controllers/VoyagerBreadController.php

```diff
@@ -5,6 +5,7 @@
 use Illuminate\Http\Request;
 use TCG\Voyager\Models\DataType;
 use TCG\Voyager\Voyager;
+use DB;
```

Comment:
It would be better to use the actual `DB` object instead of the alias, since if the user removes the alias, then this would break.